main/src/ca/uwaterloo/flix/language/phase/Stratifier.scala

```diff
@@ -59,20 +59,16 @@ object Stratifier {
 
     // Compute the stratification at every datalog expression in the ast.
     val newDefs = flix.subphase("Stratify Defs") {
-      val result = ParOps.parMap(root.defs)(kv => visitDef(kv._2)(root, g, flix).map(d => kv._1 -> d))
-      Validation.sequence(result)
+      ParOps.mapValuesFallible(root.defs)(v => visitDef(v)(root, g, flix))
```

Comment:
implicit can be omitted? (comment applies everywhere)

Reply:
If you mean `(root, g, flix)` then `root` and `g` aren't implicit.

Reply:
Oh okay.